webview/src/experiments/components/table/TableHeaderCellContents.tsx

```diff
@@ -41,9 +41,6 @@ export const ColumnDragHandle: React.FC<{
     <span
       data-testid="rendered-header"
       className={cx(styles.cellContents)}
-      onKeyDown={e => {
```

Comment:
[F] This was the cause of the breakage in the header context menu. From memory, we put this in to stop tab from causing all kinds of problems before we had a context menu. I could not find any reason to keep it around. 

@wolmir do you remember why you added this?

Reply:
@mattseddon No, I can't remember why I added it here

Sorry for the delay in the reply!